lang/de.json

```
@@ -41,6 +41,9 @@
   "Title from the viewMode GSoC branch": "Titel aus dem Ansichtsmodus GSoC-Zweig",
   "Please verify URL first": "Bitte zuerst URL bestätigen",
   "Enter a valid URL": "Geben Sie eine gültige URL ein",
+  "No URL configured. Please set URL first.": "Keine URL konfiguriert. Bitte setzen Sie zuerst die URL.",
+  "Join Organisation with QR": "Organisation mit QR beitreten",
+  "Scan QR": "QR scannen",
```

Biome (2.1.2)

[error] The key Scan QR was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)